CHANGELOG.md

```diff
@@ -8,6 +8,109 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+| [Business Partner KIT](https://eclipse-tractusx.github.io/docs-kits/category/business-partner-kit)                            |                                                                           n/a                                                                           |                                                                                                                                                        [4.0.0](https://eclipse-tractusx.github.io/docs-kits/kits/Business%20Partner%20Kit/Changelog)                                                                                                                                                         |
+| [Circularity KIT](https://eclipse-tractusx.github.io/docs-kits/category/circularity-kit)                                      |                                                                           n/a                                                                           |                                                                                                                                                   [0.2.0](https://eclipse-tractusx.github.io/docs-kits/kits/Circularity_KIT/Changelog%20Circularity%20KIT)                                                                                                                                                   |
+| [Connector KIT](https://eclipse-tractusx.github.io/docs-kits/category/connector-kit)                                          |                                                                           n/a                                                                           |                                                                                                                                            [1.2.0](https://eclipse-tractusx.github.io/docs-kits/kits/Connector%20Kit/Adoption%20View/connector_kit_adoption_view)                                                                                                                                            |
+| [Country Risk](https://github.com/eclipse-tractusx/vas-country-risk)                                                          |                       country-risk: [4.0.2](https://github.com/eclipse-tractusx/vas-country-risk/releases/tag/country-risk-4.0.2)                       |                                                                                                    Frontend: [v2.0.0](https://github.com/eclipse-tractusx/vas-country-risk/releases/tag/v2.0.0)<br/> Backend: [v2.0.0](https://github.com/eclipse-tractusx/vas-country-risk-backend/releases/tag/v2.0.0)                                                                                                     |
```

Comment:
as far as I know the separate backend repository for country risk is deprecated with this release, please remove it:
for comparison:
this release: https://github.com/eclipse-tractusx/vas-country-risk/blob/country-risk-4.0.2/.tractusx
previous release: https://github.com/eclipse-tractusx/vas-country-risk/blob/country-risk-3.0.11/.tractusx

Reply:
Well seen. Actually the Release for the [2.0.0 appVersion](https://github.com/eclipse-tractusx/vas-country-risk/blob/country-risk-4.0.2/charts/country-risk/Chart.yaml#L24) seems to be [missing](https://github.com/eclipse-tractusx/vas-country-risk/releases).

Do we have a contact person at country risk left? Fabio left AFAIK

Reply:
there's a tag available https://github.com/eclipse-tractusx/vas-country-risk/commits/v2.0.0 which triggered the image builds [backend](https://hub.docker.com/layers/tractusx/vas-country-risk-backend/2.0.0/images/sha256-8916dc20f7c0bcedab50536307b9602f9f4e534bef2fec62c08227976afdd643?context=explore) and [frontend](https://hub.docker.com/layers/tractusx/vas-country-risk/2.0.0/images/sha256-2cf81772318d04c0f3ca6cab11bb8e35ec564180653146c8b6c84fde38759356?context=explore)
Good question, who's taking over as product committer for country risk, @HeyHardy do you know that?

Reply:
Just resolved, but should be added as release. Linked to tag for now